tests/test_global_config.py

```diff
@@ -40,14 +40,14 @@ def test_settings_copy(self):
         ok_(self.config.requests['distributed_tracing'] is True)
         ok_(self.config.requests['experimental']['request_enqueuing'] is True)
 
-    def test_missing_integration(self):
+    def test_missing_integration_key(self):
```

Comment:
Why this change here in this PR? I'm not sure whether we just are no longer raising `ConfigException`.

Reply:
so... this is fun... under `nosetests` this test was never getting executed. We were ignoring any tests that matches the regex we were supplying, directories, test case classes, and test functions. (we were trying to ignore the "integration" directory).

And since the test was never running we never noticed that it wasn't testing the correct thing. It is testing that `config.unknown` will fail, but it doesn't fail anymore, we automatically add an entry for any integration we attempt to access. So instead I made it where if we try to access an integration setting that does not exist.

Reply:
Hopefully when we introduce some kind of coverage tool we'll be able to detect when our tests aren't running to avoid something like this in the future.